docs/source/docker/integration/docker_with_datasource.rst

```diff
@@ -16,12 +16,8 @@ the following workloads in one single run:
 - compute the metadata of the images
 - create a dataset in the Lightly Platform from the sampled subset
 
-Support for the following is planned but not implemented yet:
-
-- Training an embedding model is currently too slow and thus deactivated.
-- Streaming from a google cloud storage or azure storage is planned to be implemented.
-
-If you need any of these, write us so that we prioritize implementing it.
+Support for streaming from Google Cloud Storage and Azure Blob Storage is
```

Comment:
This should be rather already supported, see https://github.com/lightly-ai/lightly-core/pull/1375